src/components/Profile/ProfileComponent.jsx

```diff
@@ -24,21 +31,27 @@ import ProfileEditButtonGroup from './ProfileEditButtonGroup';
  * @param {profileComponentProps} Props - Props for ClientProfile component
  * @returns {React.JSX.Element} The UserProfile Component
  */
-const ProfileComponent = ({ clientProfile }) => {
+const ProfileComponent = ({ contactProfile }) => {
   const { session } = useSession();
   const { updateProfileInfo, setProfileData, profileData, fetchProfileInfo } =
     useContext(SignedInUserContext);
 
+  // Public Profile Data
   const [profileName, setProfileName] = useState(profileData?.profileInfo?.profileName);
   const [nickname, setNickname] = useState(profileData?.profileInfo?.nickname);
```

Comment:
around here...

we have `profileData`, then either `profileInfo` or `privateProfileData`.   Could we have these consistent?  I'm assuming `profileInfo` is actually the public profile data, but could be wrong.

I guess it doesn't matter, but I would personally prefer consistency of something like:
`publicProfileInfo` and `privateProfileInfo` or the word data instead.

Reply:
oh we also have `profileData` and `profileDataSolid`, maybe same idea with:
`profileDataPass` and `profileDataSolid`

Reply:
just a thought, not the biggest deal

Reply:
Yes, `profileInfo`, `profileData` are both related to public info and `privateProfileInfo` and `privateProfileData` are both related to private info, see `fetchProfileInfo` in `src/model-helpers/Profile.js`. Public and private profile information are stored separately with distinction being the `private` prefix.

As for `profileDataSolid`, that is the updated data fetched from user's Pod in case of updates ad it pulls all data from Solid public and private hence `profileDataSolid`.

Reply:
Alright! Then I approve the way it is 😅